core/src/test/java/org/infinispan/configuration/parsing/UnifiedXmlFileParsingTest.java

```diff
@@ -71,6 +70,8 @@
 @Test(groups = "unit", testName = "configuration.parsing.UnifiedXmlFileParsingTest")
 public class UnifiedXmlFileParsingTest extends AbstractInfinispanTest {
 
+   public static final String TMPDIR = System.getProperty("java.io.tmpdir");
```

Comment:
Shouldn't we be using `CommonsTestingUtil.tmpDirectory`?

Reply:
No, because this test is only about ensuring that it is parsing things correctly (including expanding of system properties). It won't actually use the persistent paths for anything